docs/btb.selection.rst

```diff
@@ -0,0 +1,86 @@
+btb.selection package
```

Comment:
I think this file as well as `btb.submodule.rst` for any submodule do not need to be committed to the repo, because they are autogenerated by sphinx every time the docs are built.

Reply:
I don't see why they should be kept back.
They will only change when the corresponding code is changed, and committing the docs alongside the code seems a good practice to me.
Also, even though this is just a minor side effect, having the rst docs committed allows browsing them directly in github without having to generate them locally or publishing them somewhere else.

Reply:
I guess it doesn't matter if they are included or not. 

In my opinion, it doesn't help much to look through the auto-generated module documentation source files. The reason is that it is much better to browse the rendered documentation :) This is something that Travis will do automatically when we merge to master. For example, compare https://hdi-project.github.io/fhub_core/installation.html (there IS a source file in docs/installation.rst on master) and https://hdi-project.github.io/fhub_core/modules.html (there is NO source file because it is automatically generated and pushed to gh-pages branch).

Reply:
Good point. I'll merge it as it is right now, to avoid keeping the PR stuck, and later on we can remove the extra docs.
We could also include a `docs/btb.*.rst` entry in `.gitignore`